vas2nets/tasks.py

```diff
@@ -0,0 +1,29 @@
+import csv
+import requests
+
+from django.conf import settings
+from celery.task import Task
+
+from .models import VoiceCall
+
+
+class FetchVoiceData(Task):
+
+    def get_data(self, date):
+        url = "%s?report_date=%s" % (settings.V2N_VOICE_URL, date)
+
+        content = requests.get(url).content.decode('utf-8')
+
+        return list(csv.reader(content.splitlines(), delimiter=','))[1:]
+
+    def run(self, date, **kwargs):
+        data = self.get_data(date)
+        for row in data:
+            VoiceCall.objects.create(
+                created_at=row[0],
```

Comment:
We probably want to add timezone data to this datetime.

Can we figure out what timezone they're sending in the CSV, and add that data to the datetime object before storing it in the DB?